doc/edit.md

```diff
@@ -128,30 +128,30 @@ The project already has `generics-sop` and `sop-core` added to the .cabal file,
 > import Generics.SOP
 ```
 
-We also have [the `these` package](https://hackage.haskell.org/package/these-1.1.1.1/docs/Data-These.html) added to the .cabal file because it provides the above `These` type from `Data.These` module. To explore the SOP representation of the `These` type, let us do some bootstrapping:
+We also have [the `these` package](https://hackage.haskell.org/package/these-1.1.1.1/docs/Data-These.html) added to the .cabal file, because it provides the above `These` type from `Data.These` module. To explore the SOP representation of the `These` type, let's do some bootstrapping:
 
 ```haskell
 > import Data.These 
 > instance Generic (These a b) -- Derive generics-sop instance
 > let breakfast = These "Egg" "Sausage" :: These String String
 ```
 
-We derived `Generic` on the type and created a term value called `breakfast` (we are eating both eggs and sausages). To get the SOP representation of this value, we can use [`from`](https://hackage.haskell.org/package/generics-sop-0.5.1.2/docs/Generics-SOP.html#v:from):
+We derived `Generic` on the type and created a term value called `breakfast` (for which we are eating both eggs and sausage). To get the SOP representation of this value, we can use [`from`](https://hackage.haskell.org/package/generics-sop-0.5.1.2/docs/Generics-SOP.html#v:from):
```

Comment:
It doesn't have to be. But the reason I made it singular is because it was singular in all of your examples and tables. I made it consistent to avoid confusion.